analysis/report_templates/default.html

```diff
@@ -232,17 +232,51 @@ <h5 class="center-align">Mann-Whitney U test</h4>
                 </li>
             </ul>
 
-            {% if coverage_report %}
+            {% if detailed_coverage_report %}
+            <ul class="collapsible">
+                <li>
+                    <div class="collapsible-header">
+                        Differential matrix
+                    </div>
+                    <div class="collapsible-body">
+
+                        <div class="row">
+                            <div class="col s6 offset-s3">
+                                <h5 class="center-align">Ranking by unique edges covered</h4>
+                                    <img class="responsive-img materialboxed"
+                                         src="{{ benchmark.unique_region_ranking_plot }}">
+                                    The table shows the unique edges coverage rank of each fuzzer
+                                    on this benchmark. Unique edges are edges only covered by one
+                                    certain fuzzer.
+                            </div>
+                        </div> <!-- row -->
+
+                        <div class="row">
+                            <div class="col s6 offset-s3">
+                                <h5 class="center-align">Complementary pairs table</h4>
+                                    <img class="responsive-img materialboxed"
+                                         src="{{ benchmark.complementary_pairs_plot }}">
+                                    This table is a square matrix where each row and column
+                                    represents a fuzzer, and each cell contains a number showing the
+                                    number of edges covered by the fuzzer of the column but not by
+                                    the fuzzer of the row.
```

Comment:
Consider just saying something like:

`Each cell represents the number of edges covered by the fuzzer of the column but not by the fuzzer of the row.`

Also, what would be useful if we added "axis titles" to the side and bottom of the table (next and below the fuzzer names), saying  something like "covered by", "not covered by". That'd make reading it easier, as one doesn't need to think about which is the fuzzer "of the row" and "of the column". wdyt?

Reply:
Very good point! I'll add it now.